### ISSUE TITLE
Visualizing specific layers Tensorboard Callback

### ISSUE THREAD

**Issue by rodyt** [tensorflow/tensorboard#3207]
Keras's Tensorboard callback has a `write_images` and a `histogram_freq` argument that allows the weights and activations to be saved to Tensorboard to visualize during training. This is helpful for visualizing training in real time.

The issue is: this saves the information for every layer and makes Tensorboard very messy to read, especially if I have logged other images to Tensorboard. This can be seen in the images below:

![Images2](https://user-images.githubusercontent.com/36773880/73596541-aa0f2c80-44d7-11ea-8d40-e3e8fda9c4be.png)
![Images1](https://user-images.githubusercontent.com/36773880/73596542-aa0f2c80-44d7-11ea-9cd7-23dbf675e9fd.png)

A lot of this logged information is redundant.

Is there any way to make the weight distribution and activation visualizations more organized so that they don't fill up my whole screen?

Is there any way to only visualize certain layers using the Keras callback? (i.e. I don't want to visualize the BatchNorm layers)

**Comment by stephanwlee** [comment]
The Keras callback is owned by the Keras team so you will have better luck filing bug against them. 

One way to mitigate the issue, as you may already know, is to use tags filter on the top of the page to find/filter the layers you want to inspect.
